rpms/atomic-openshift-odo.spec

```diff
@@ -0,0 +1,77 @@
+#this is a template spec and actual spec will be generated
+#debuginfo not supported with Go
+%global debug_package %{nil}
+%global package_name atomic-openshift-odo
```

Comment:
I have not reviewed any spec files in the past. Heck, I don't even understand them so please pardon me if the query is silly/naive.

Does this mean that `yum install atomic-openshift-odo` will install the `odo` binary for the users? If yes, out of sheer curiosity, why do we have `atomic` in the name? 

If no, what does this mean? :smile:

Reply:
> I have not reviewed any spec files in the past. Heck, I don't even understand them so please pardon me if the query is silly/naive.
> 
> Does this mean that `yum install atomic-openshift-odo` will install the `odo` binary for the users? If yes, out of sheer curiosity, why do we have `atomic` in the name?
>
> If no, what does this mean?
yes and no both. It is actually prefixed to package name
full name = package_name-odo_version-odo_release.dist.arch.rpm
the atomic-openshoft is rec prefix for all openshift packages

Reply:
> Does this mean that yum install atomic-openshift-odo will install the odo binary for the users?

Yes.

Its upto us to decide what we want to name, the naming discussions are always ..
but we can align to what's done for `oc`, `oc` is provided by `openshift-clients` RPM, which is a RPM of `openshift` package (multiple RPMs, part of one package).

Reply:
> > Does this mean that yum install atomic-openshift-odo will install the odo binary for the users?
> 
> Yes.
> 
> Its upto us to decide what we want to name, the naming discussions are always ..
> but we can align to what's done for `oc`, `oc` is provided by `openshift-clients` RPM, which is a RPM of `openshift` package (multiple RPMs, part of one package).

Thanks @navidshaikh for the response. It helps. And I agree that naming discussions are always... I'm afraid I'm the one who's making it painful this time. :joy:

@kadel @mohammedzee1000 do we really need `atomic` in the name of the rpm that provides `odo` binary? I feel that `openshift-odo` is good enough. I don't see why we want to connect this with Project Atomic which is, AFAIU, more about underlying container technologies.

Reply:
well @kadel i will leave final call to you. But i would require the decision at earliest as i will need to raise rcm ticket to set up a new brew package (removing old one in the process)

Reply:
It should not matter too much in my book as the rpm is for rcm satisfaction and the paper trail. We will be delivering it to other delivery mechanisms such as mirror / cdn / cli artifacts image

Reply:
If it is possible I would drop the `atomic` prefix.